doc/ext/convert-svg-to-pdf.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+"""
+    Converts SVG images to PDF using chrome in case the builder does not
+    support SVG images natively (e.g. LaTeX).
+
+"""
+import subprocess
+
+from sphinx.errors import ExtensionError
+from sphinx.locale import __
+from sphinx.transforms.post_transforms.images import ImageConverter
+from sphinx.util import logging
+from errno import ENOENT, EPIPE, EINVAL
+import os
+import platform
+import tempfile
+
+if False:
+    # For type annotation
+    from typing import Any, Dict  # NOQA
+    from sphinx.application import Sphinx  # NOQA
+
+
+logger = logging.getLogger(__name__)
+
+
+class Converter(ImageConverter):
+    conversion_rules = [
+        ('image/svg+xml', 'application/pdf'),
+    ]
+
+    def is_available(self):
+        # type: () -> bool
+        """Confirms if converter is available or not."""
+        return True
+
+    def chrome_command(self):
```

Comment:
This function only dispatches by different OS, but it should provide a usable command like the deleted script did. And if the command isn't available, it should exit with an error meesage, instead of a warning and proceed.
Moreover, I don't think we should force users to install proprietary Google Chrome. At least `chromium` command should be tried first of all.

Reply:
I am not sure if all chromiums are able to perform this task in headless mode. @asmeurer implemented this thing in shell script and I only converted it into python script to run on any OS

> implemented this thing in shell script and I only converted it into python script to run on any OS
I changed logger.warning to logger.error

Reply:
I didn't test chromium (I only did minimal testing on Linux). If it can work, I agree we should try it too. My guess is that it will work since the SVG code shouldn't be part of the Google specific parts of Chrome.